saltyrtc/__init__.py

```diff
@@ -3,22 +3,29 @@
 """
 import itertools
 
-from .exception import *
-from .common import *
-from .message import *
-from .protocol import *
-from .client import *
-from .server import *
-from . import util
+# noinspection PyUnresolvedReferences
+from .exception import *  # noqa
+# noinspection PyUnresolvedReferences
+from .common import *  # noqa
+# noinspection PyUnresolvedReferences
+from .message import *  # noqa
+# noinspection PyUnresolvedReferences
+from .protocol import *  # noqa
+# noinspection PyUnresolvedReferences
+from .client import *  # noqa
+# noinspection PyUnresolvedReferences
+from .server import *  # noqa
+# noinspection PyUnresolvedReferences
+from .util import *  # noqa
```

Comment:
Might be worse now that I've moved it into a separate package `saltyrtc.server`. However, I can't change that as I cannot change the lines in `saltyrtc/__init__.py`.